### ISSUE TITLE
CTest tests don't run in parallel from the testing pane

### ISSUE THREAD

**Issue by dgazzoni** [microsoft/vscode-cmake-tools#3322]
As originally reported in #3091, CTest aren't running in parallels when started from the testing pane, using the button "Run tests" with the double-play-like icon.

In #3091 this was fixed if run by pressing "Run CTest" in the status bar, and possibly through other paths, but specifically not through the testing pane. As instructed by @snehara99 I'm opening a new issue to address this specific use case.

**Comment by gcampbell-msft** [comment]
@snehara99 Was this resolved by #3271 ?

**Comment by snehara99** [comment]
> @snehara99 Was this resolved by #3271 ?

No, we allowed the parallel tests from the status bar and the Project Status View, but not from the Test Explorer yet. We decided to approach that separately as a feature.

**Comment by geertj** [comment]
Commenting on this issue since the others are closed. Where is the option "Run CTest" in the status bar? Recently items have moved to the CMake pane on the left, and I don't see this option either in the CMake pane or in the status bar.

**Comment by hippo91** [comment]
@geertj  thanks for the hint! However in my case the tests, even when launched with your method, are not run in parallel. Have you got the same pb with latest vscode version (vscode version: 1.86.0, cmake tools version: v1.16.32)?

**Comment by hippo91** [comment]
@geertj  i finally succeed in but i had to upgrade to the pre-release version of cmake tools and then add explicitly the number of jobs in the `cmake.ctest` default ctest args settings.

**Comment by hippo91** [comment]
@gcampbell-msft i think the label `fixed (release pending)` can be added to this issue.

**Comment by v-frankwang** [comment]
@hippo91 Thank you very much for your reply! This label: fixed (release pending) has been added by gcampbell.
